src/lib/crypto/kimchi_backend/gadgets/ecdsa.ml

```diff
@@ -18,9 +18,8 @@ let tuple6_of_array array =
  *   Scalar field external checks:
  *     Bound checks:         6
  *     Canonical checks:     2
- *     Multi-range-checks:   2
  *     Compact-range-checks: 2
- *     Total range-checks:   42
+ *     Range-checks:         36
```

Comment:
HA! Sooo... our initial feeling that we might eventually be doing less rows might be true???

Reply:
Well, overall the verification tests showed the size of the circuits increased significantly.  Perhaps we can remove some of the canonical checks, but we should be cautious and I'm not certain this optimizations are necessary for the POC.